ui.frontend/src/view/FormCheckBox.js

```diff
@@ -0,0 +1,60 @@
+import FormPanel from "./FormPanel";
+
+/*******************************************************************************
+ * Copyright 2023 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+import FormFieldBase from "./FormFieldBase.js";
+
+/**
+ * Class representing components baed on CheckBox.
+ * @extends module:FormView~FormPanel
```

Comment:
Please fix the js doc. Also why wasn't this part of the PR raised by @barshat7

Reply:
This file was added later to have a common _setModel_ and _updateValue_ for switch and checkbox